tools/integration_tests/emulator_tests/emulator_tests.sh

```diff
@@ -19,6 +19,13 @@ set -eo pipefail
 # Display commands being run
 set -x
 
+architecture=$(dpkg --print-architecture)
+if [ $architecture == "arm64" ];then
+  # TODO: Remove this when we have an ARM64 image for the storage test bench.
```

Comment:
Please add internal bug ID to track this

Reply:
Done